src/Tribe/Status/Manager.php

```diff
@@ -0,0 +1,179 @@
+<?php
+
+
+/**
+ * Class Tribe__Tickets__Status__Manager
+ *
+ * @since TBD
+ */
+class Tribe__Tickets__Status__Manager {
+
+	/**
+	 * Active Modules
+	 *
+	 * @var array
+	 */
+	protected $active_modules;
+
+	/**
+	 * An array of status objects for WooCommerce Tickets
+	 *
+	 * @var array
+	 */
+	protected $status_managers = array(
+		//'EDD' => 'Tribe__Tickets_Plus__Commerce__WooCommerce__Status_Manager',
+		//'RSVP' => 'Tribe__Tickets_Plus__Commerce__WooCommerce__Status_Manager',
+		//'Tribe Commerce' => 'Tribe__Tickets_Plus__Commerce__WooCommerce__Status_Manager',
+		'WooCommerce' => 'Tribe__Tickets_Plus__Commerce__WooCommerce__Status_Manager',
```

Comment:
I know I saw this before, but I just thought of it again. Perhaps we should use the commerce provider slugs that we have standardized within the ET suite?

* `woo`
* `edd`
* `rsvp`
* `tribe-commerce`